src/modules/universal-discovery/module-usage.md

```diff
@@ -81,28 +81,41 @@ Without all the following properties the Universal Discovery module will not wor
 Optionally, Universal Discovery module can take a following list of props:
 
 **loadContentInfo** _{Function}_ - loads content info. It takes 3 params: `restInfo`, `contentId` and `callback`
+
```

Comment:
This file should be removed anyway. We have even PR for this, so not needed changes.